tdm-be/src/controller/config.ts

```diff
@@ -21,15 +21,7 @@ const router = express.Router();
  *           type: array
  *           items:
  *             $ref: '#/components/schemas/SwaggerApi'
- *           example: [{'url':'https://data-computer.services.istex.fr', 'retrieveUrl':'/v1/retrieve', 'tags':[{'name':'data-computer','excluded':['/v1/collect','/v1/retrieve','/v1/mock-error-async','/v1/mock-error-sync']}]}]
- *         mailSuccess:
- *           type:
- *             $ref: '#/components/schemas/Mail'
- *           example: {'subject':'Objet du mail succès', 'text':'Vous pouvez télécharger le fichier enrichi à l&apos;adresse ci-dessous' }
- *         mailError:
- *           type:
- *             $ref: '#/components/schemas/Mail'
- *           example: {'subject':'Objet du mail d&apos;erreur', 'text':'Une erreur s&apos;est produite lors de l&apos;enrichissement' }
+ *           example: [{'url':'https://data-computer.services.istex.fr', 'retrieveUrl':{url: '/v1/retrieve-csv', fileExtension: 'csv'}, 'tags':[{'name':'data-computer','excluded':['/v1/collect','/v1/retrieve','/v1/mock-error-async','/v1/mock-error-sync']}]}]
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**nitpick (non-blocking):** ‏I don't see any modification in the program itself, to use `retrieveUrl.url` instead of `retrieveUrl`...

Reply:
This has already been merge in previous PR https://github.com/Inist-CNRS/tdm-factory/blob/a3abd6075655b9f06d10b355eea3b9ec9c09cf9c/tdm-be/src/worker/enrichmentHook.ts#L88